pylzy/tests/scenarios/fully_cached_graph/expected_stdout

```diff
@@ -0,0 +1,3 @@
+foo was called
+bar was called
+message from bar: number is 42
```

Comment:
where is the message from the second execution?

Reply:
execution graph was cached while first execution